tests/tests.js

```diff
@@ -317,3 +317,49 @@ QUnit.test("Control structures don't eval depth-first", function(assert) {
   jsonLogic.apply({"or": [{"push": [true]}, {"push": [true]}]});
   assert.deepEqual(i, [true]);
 });
+
+
+
+
```

Comment:
Added a few tests to ensure that the individual JSONLogic instances weren't accidentally sharing state.